app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by(email: params[:email])
```

Comment:
em có thể lượt bỏ ngoặc được nè
Phân biệt giúp chị `@user` và `user` có gì khác nhau

Reply:
user là biến cục bộ chỉ có giá tri tại nơi khai báo.
@user là biến của đối tượng phạm vi là toàn class.